test/fixtures/items_sample.csv - items.csv.csv

```diff
@@ -0,0 +1,222 @@
+id,name,description,unit_price,merchant_id,created_at,updated_at

```

Comment:
🔪  Don't commit flat data to your repository if you don't have to. Its a bad habit, especially if you work in an environment where the data is sensitive, as in health care or finance.